lib/TelemetryLog.js

```diff
@@ -29,12 +33,12 @@ class TelemetryLog {
     });
   }
   showPanel(domain) {
-    this.panelShowCount += 1;
+    simpleStorage.storage.TelemetryLogData.panelShowCount += 1;
```

Comment:
To cut down on boilerplate, you can write setters/getters for this:

```js
get logData() {
  return simpleStorage.storage.TelemetryLogData;
}

get panelShowCount() {
  return this.logData.panelShowCount;
}

set panelShowCount(newCount) {
  this.logData.panelShowCount = newCount;
}
```

I haven't tested that, but I _think_ it should work. The [docs on getters](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/get) will probably be helpful.